usvm-core/src/main/kotlin/org/usvm/memory/Memory.kt

```diff
@@ -26,11 +27,11 @@ interface UMemoryRegionId<Key, Sort : USort> {
 }
 
 interface UReadOnlyMemoryRegion<Key, Sort : USort> {
-    fun read(key: Key): UExpr<Sort>
+    fun read(key: Key, ownership: MutabilityOwnership): UExpr<Sort>
```

Comment:
Are there any cases when we need ownership in a read operation? It seems we can always use default ownership

Reply:
Yes, reading by brand new `SymbolicCollectionId` modifies persistent maps in `URefMapRegion`, `UMapRegion`, etc

Reply:
I think in such cases we can still use default ownership